python/cudf/cudf/core/frame.py

```diff
@@ -3673,6 +3673,12 @@ def ceil(self):
         3    5.0
         dtype: float64
         """
+
+        warnings.warn(
+            "Series.ceil is deprecated and will be removed in the future",
```

Comment:
If applied here, both `Series`/`DataFrame` `ceil`/`floor` function will show this warning message.

```suggestion
            "Series and DataFrame.ceil is deprecated and will be removed in the future",
```

Reply:
Same goes to floor below.